<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KitPhone</class>
 <widget class="QMainWindow" name="KitPhone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>PS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>SK</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>41</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>86</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>connect a2a</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>151</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>drswinghead</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>86</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>call pstn</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>80</y>
           <width>201</width>
           <height>25</height>
          </rect>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>9086013567890</string>
          </property>
         </item>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>301</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>连接skype</string>
         </property>
        </widget>
        <widget class="QToolButton" name="toolButton">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>10</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>ST</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFsdfdsf">
    <property name="title">
     <string>fsdfdsf</string>
    </property>
    <addaction name="actionDsfdsfdsf"/>
   </widget>
   <widget class="QMenu" name="menuXcvogrre">
    <property name="title">
     <string>xcvogrre</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu8u3298rdsf">
    <property name="title">
     <string>8u3298rdsf</string>
    </property>
   </widget>
   <addaction name="menuFsdfdsf"/>
   <addaction name="menuXcvogrre"/>
   <addaction name="menu8u3298rdsf"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDsfdsfdsf">
   <property name="text">
    <string>dsfdsfdsf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
